stock-tracker/src/store/store.js

```diff
@@ -0,0 +1,20 @@
+import { createStore, applyMiddleware, compose } from "redux";
+import { rootReducer } from "./rootReducer";
+import { initialStartupMiddlware } from "./initialStartupMiddleware";
+import { searchMiddleware } from "../features/search/redux/searchMiddleware";
+import { chartMiddleware } from "../features/chart/redux/chartMiddleware";
+import { socketService } from "../socket";
+
+const middlwares = [
```

Comment:
sp: `middleware`. It's a singulare tantum.

Reply:
You're still missing the `e`

Reply:
oops